app/templates/categories/benefits/appeal.html

```diff
@@ -0,0 +1,46 @@
+{# This page was built as a prototype #}
```

Comment:
Is this line about the prototype still needed?

Reply:
Removed